src/quo/components/colors/color/style.cljs

```diff
@@ -1,24 +1,33 @@
 (ns quo.components.colors.color.style
   (:require
+    [quo.components.colors.color.constants :as constants]
     [quo.foundations.colors :as colors]))
 
-(def color-button-common
-  {:width             48
-   :height            48
-   :border-width      4
-   :border-radius     24
-   :margin-horizontal 4
-   :transform         [{:rotate "45deg"}]
-   :border-color      :transparent})
+(defn color-button-common
+  [window-width]
+  {:width         48
+   :height        48
+   :border-width  4
+   :border-radius 24
+   :margin-right  (-> window-width
+                      (- constants/IPHONE_11_PRO_VIEWPORT_WIDTH)
```

Comment:
I think this deserves a comment - or put this in a private function to help explain the idea/logic here 👍

Reply:
Done